include/seqan3/alignment/pairwise/edit_distance_unbanded.hpp

```diff
@@ -139,15 +120,14 @@ class pairwise_alignment_edit_distance_unbanded
     //!\brief When true the computation will use the ukkonen trick with the last active cell and bounds the error to config.max_errors.
     static constexpr bool use_max_errors = detail::max_errors_concept<align_config_t>;
     //!\brief Whether the alignment is a semi-global alignment or not.
-    static constexpr bool is_semi_global = detail::semi_global_config_concept<align_config_t>;
+    static constexpr bool is_semi_global = traits_t::is_semi_global_type::value;
     //!\brief Whether the alignment is a global alignment or not.
-    static constexpr bool is_global = detail::global_config_concept<align_config_t> && !is_semi_global;
+    static constexpr bool is_global = !is_semi_global;
 
     //!\brief How to pre-initialize hp.
     static constexpr word_type hp0 = is_global ? 1 : 0;
 
     static_assert(8 * sizeof(word_type) <= 64, "we assume at most uint64_t as word_type");
-    static_assert((is_global && !is_semi_global) || (!is_global && is_semi_global), "Either set global or semi-global");
```

Comment:
why did you remove this? Can I do both at the same time?

Reply:
it is not possible to have set both at the same time with the new logic so the assertion is always true and therefore redundant.